frontend/src/queries/example.ts

```diff
@@ -0,0 +1,62 @@
+import { query2dcids, Query, CategoryType } from "./query"
+import { Dcid } from "./dcids"
+
+import { DCIDS } from "./example_dcids"
+
+
+const Categories = {
+    'gender': new Set<string>([
+        'Male',
+        'Female',
+    ]),
+    'ethnicity': new Set<string>([
+        'HispanicOrLatino',
```

Comment:
Due to a DataCommons nuance, 'ethnicity' includes only 'HispanicOrLatino', and everything else is 'race'. Again on line 57

Reply:
My category names, 'gender', 'ethnicity', are my invention. I checked the keys and `HispanicOrLatino` appears to behave as if it's in the same category as other 'race` dimensions. Do we need to confirm that the `HispanicOrLatino` category/dimension is a synonym for the 'race' dimension before including it? I'm not seeing both 'race' and 'ethnicity' in any of the listed DCIDs.